cmd/noms/noms_ds_test.go

```diff
@@ -25,21 +26,25 @@ type nomsDsTestSuite struct {
 
 func (s *nomsDsTestSuite) TestEmptyNomsDs() {
 	dir := s.DBDir
+	dbName := dir+"/name"
 
-	cs := nbs.NewLocalStore(dir+"/name", clienttest.DefaultMemTableSize)
+	os.MkdirAll(dbName, 0777)
+	cs := nbs.NewLocalStore(dbName, clienttest.DefaultMemTableSize)
 	ds := datas.NewDatabase(cs)
 
 	ds.Close()
 
-	dbSpec := spec.CreateDatabaseSpecString("nbs", dir+"/name")
+	dbSpec := spec.CreateDatabaseSpecString("nbs", dbName)
 	rtnVal, _ := s.MustRun(main, []string{"ds", dbSpec})
 	s.Equal("", rtnVal)
 }
 
 func (s *nomsDsTestSuite) TestNomsDs() {
 	dir := s.DBDir
+	dbName := dir+"/name"
```

Comment:
Same Q as above. Unclear why the (admittedly already-existing) code makes a new directory. Do we need that?